samples/react-chat/src/App.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { memo, useCallback, useState } from 'react';
+import React, { memo, useCallback, useState, useEffect } from 'react';
 import './App.css';
 import { Get, Login } from '@microsoft/mgt-react';
 import { Chat, NewChat } from '@microsoft/mgt-chat';
+import { CacheService, log } from '@microsoft/mgt-element';
```

Comment:
I don't think that these imports are used at present.

Reply:
Correct, removed.